src/runtime-rs/crates/resource/src/volume/mod.rs

```diff
@@ -79,7 +79,7 @@ impl VolumeResource {
                 Arc::new(
                     block_volume::BlockVolume::new(d, m, read_only, cid, sid)
                         .await
-                        .with_context(|| format!("new share fs volume {:?}", m))?,
+                        .with_context(|| format!("new block volume {:?}", m))?,
```

Comment:
This typo fix clearly doesn't belong to this commit, which is large enough already. Please drop it.

Reply:
Fixed